rom-elks.c

```diff
@@ -381,11 +382,11 @@ static int int_13h ()
 
 		default:
 			printf ("fatal: INT 13h: AH=%hxh not implemented\n", ah);
-			assert (0);
+			return err;
```

Comment:
No return here, please, but a break for structured programming.

Reply:
It is there because in this case, a -1 return is required.  I understand the structured programming, but this particular EMU86 interrupt handler requires a 0 return on implemented functions, regardless of whether there was a disk error or not (CF is for that).

I have a fix for #15, which uses the same approach: default case returns -1, otherwise 0. I don't think is is bad for these few cases, but you're welcome to rewrite how you want. I'm just trying to get ELKS to boot again so I can continue implementing functions.